stac-example/src/main/scala/geotrellis/stac/raster/StacAssetRasterSource.scala

```diff
@@ -0,0 +1,44 @@
+package geotrellis.stac.raster
+
+import com.azavea.stac4s.StacItemAsset
+import geotrellis.proj4.CRS
+import geotrellis.raster.io.geotiff.OverviewStrategy
+import geotrellis.raster._
+import geotrellis.vector.Extent
+import _root_.io.circe.JsonObject
+import cats.syntax.option._
+
+case class StacAssetRasterSource(
+  asset: StacItemAsset,
+  itemProperties: JsonObject = JsonObject.empty,
+  private[raster] val underlyingRS: Option[RasterSource] = None,
+  private[geotrellis] val targetCellType: Option[TargetCellType] = None
+) extends RasterSource {
+  private lazy val underlying = underlyingRS.getOrElse(RasterSource(asset.href.replace("https://s3-us-west-2.amazonaws.com/landsat-pds/c1/", "gdal+file:///Users/daunnc/Downloads/lc8-test/")))
```

Comment:
Don't forget to remove the local ref here.

Reply:
woooot thanks!